src/services/school/hooks/index.js

```diff
@@ -73,14 +73,15 @@ const decorateYears = async (context) => {
 		school.years = facade.toJSON();
 	};
 	try {
+		const resultData = context.result.data || context.result;
 		switch (context.method) {
 			case 'find':
-				context.result.data.forEach((school) => {
+				resultData.forEach((school) => {
 					addYearsToSchool(school);
 				});
 				break;
 			case 'get':
-				addYearsToSchool(context.result);
+				addYearsToSchool(resultData);
```

Comment:
in case of 'get' context.result will always be one school, so this change here is not necessary, or even harmful if there was an attribute called data in a school object

Reply:
Ok. Thanks